docs/source/deployment/heroku.md

```diff
@@ -97,7 +97,7 @@ Some things to note:
 
     Alternatively, you can [configure environment variables](./heroku/#configuring-environment-variables) through the Heroku dashboard.
 
-- Remember that [GraphQL Playground](../testing/graphql-playground) is disabled by default when Apollo Server is in a production environment.
+- Remember that introspection is disabled by default when Apollo Server is in a production environment, which prevents tools like Apollo Sandbox from working.
```

Comment:
I don't know why the dead link checker wasn't choking before on this one...

Reply:
I think when I re-targeted the community PR to this branch, it didn't re-run link checks